README.md

```diff
@@ -128,23 +168,72 @@ impl FutureService for HelloServer {
         Ok(format!("Hello, {}!", name))
     }
 }
+```
+
+In this example, `hello` returns `Result`, but in more complex servers, you might talk to another
+server or do something using IO. In those cases, the `HelloFut` type would be something more
+complex. An example of such a server can be found in `examples/two_servers.rs`. In cases where RPCs
```

Comment:
Not sure about calling this "RPCs". To me, RPC implies the execution of a remote procedure call (client side). Maybe this should be "RPC handlers" so it's clear that we're talking about the server execution.